languages/jelly/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:buster
+
+WORKDIR /tmp
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+        wget \
+        python3 \
+        python-pip3 \
+        && \
+    wget https://github.com/DennisMitchell/jellylanguage/archive/master.tar.gz && \
```

Comment:
I'd recommend against just downloading from `master`, even though Jelly isn't actively updated any more. Pin it to a specific commit (e.g. `70c9fd93ab009c05dc396f8cc091f72b212fb188.tar.gz`)